src/wtr-config.js

```diff
@@ -0,0 +1,106 @@
+import { defaultReporter } from '@web/test-runner';
+import { playwrightLauncher } from '@web/test-runner-playwright';
+import { visualDiff } from './visual-diff-plugin.js';
+import { visualDiffReporter } from './visual-diff-reporter.js';
+
+const DEFAULT_PATTERN = type => `./**/*.${type}.js`;
```

Comment:
Probably 95% of repos (or more) use `./test/**/*.test.js` as the pattern, just not core.